### DISCUSSION TITLE
DJR plot

### DISCUSSION THREAD

**Discussion by semlalisouad** [MadAnalysis/madanalysis5#128] [General]
Dear MAdanalysis team,

I used MG to generate events for DY process with 4jets (pp > mu+mu- jj b b~) with MLM, with
qcut = 19, xqcut = 10, ickkw =1.
Please find attached the DY_banner.txt. 

Merging scale, Cross-section [pb] , MC uncertainty [pb] 
19,   2.167654e+00,   1.10e-02            
28.5,  8.429754e-01,  6.86e-03            
38,  4.335304e-01,  4.92e-03 

I am trying to generate DJR plots using madanalysis, but the plots are empty. 
import /home/semlali/MC/MG5_aMC_v2_9_5/DY2b2jets/Events/run_05/DY_pythia8_events.hepmc.gz
set main.merging.check = true
set main.merging.ma5_mode = true
set main.merging.njets = 4
Am I missing something?

Many thanks for your help

Best
Souad
[run_05_DY_banner.txt](https://github.com/MadAnalysis/madanalysis5/files/9542049/run_05_DY_banner.txt)

**Comment by jackaraz** [comment]
Hi @semlalisouad i only see one sample in your event, to work with djr you need events with different jet multiplicities and i don’t mean the ones generated during shower. Generally we generate 0, 1, 2 jet events and plot djr distributions.

**Comment by semlalisouad** [comment]
Hi Jack,

Thank you for your reply. I also tried to generate the process up to 4-jets, but even in that case, the DJR plots are empty. 
Please find attached the banner.txt (0,1,2,3,4 jets). 

Thank you for your help. 

Best
Souad

[run_01_DY_banner.txt](https://github.com/MadAnalysis/madanalysis5/files/9542125/run_01_DY_banner.txt)
